keda/2.12.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,10 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: alpha
+  operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
It was time ago since I did OLM, but if memory serves me correct, you should change this from alpha to staging as well.

Reply:
Good catch, thanks!